StarkInfra/StarkInfra/CardMethod/CardMethod.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using StarkInfra.Utils;
+
+
+namespace StarkInfra
+{
+    /// <summary>
+    /// CardMethod object
+    /// <br/>
+    /// CardMethod's codes are used to define methods filters in IssuingRules.
+    /// <br/>
+    /// Properties:
+    /// <list>
+    ///     <item>Code [string]: method's code. Options: "chip", "token", "server", "manual", "magstripe", "contactless"</item>
+    ///     <item>Name [string]: method's name. ex: "token"</item>
+    ///     <item>Number [string]: method's number. ex: "81"</item>
+    /// </list>
+    /// </summary>
+    public partial class CardMethod : SubResource
+    {
+        public string Code { get; }
+        public string Name { get; }
+        public string Number { get; }
+
+        /// <summary>
+        /// CardMethod object
+        /// <br/>
+        /// CardMethod's codes are used to define methods filters in IssuingRules.
+        /// <br/>
+        /// Parameters (required):
+        /// <list>
+        ///     <item>code [string]: method's code. Options: "chip", "token", "server", "manual", "magstripe", "contactless"</item>
+        /// </list>
+        /// Attributes (return-only):
+        /// <list>
+        ///     <item>name [string]: method's name. ex: "token"</item>
+        ///     <item>number [string]: method's number. ex: "81"</item>
+        /// </list>
+        /// </summary>
+        public CardMethod(string code, string name = null, string number = null)
+        {
+            Code = code;
+            Name = name;
+            Number = number;
+        }
+        
+        /// <summary>
+        /// Retrieve CardMethods
+        /// <br/>
+        /// Receive a generator of CardMethod objects in the Stark Infra API
```

Comment:
Bom deixar no padrão que está em bank.

`Receive an IEnumerable of CardMethod objects previously created in the Stark Infra API`

Se der um search all em "generator" acho que agiliza.